packages/hapi/modules/plugin/plugin.js

```diff
@@ -37,13 +37,13 @@ const createPlugin = ({ options } = {}) => {
   const plugin = {
     name: pkg.name,
     version: pkg.version,
-    register(server) {
+    register(server, o, done) {
```

Comment:
Please check master. This is defaulted now as I can see no reference on hapi passing it in. Can you clarify?

Reply:
This will break the plugin. `done` is not passed and invoking it below makes any server fail to start.